src/python/ddapp/tasks/robottasks.py

```diff
@@ -706,6 +708,15 @@ def run(self):
         side = self.properties.getPropertyEnumValue('Side').lower()
         self.getHandDriver(side).sendCustom(self.properties.getProperty('Amount'), 100, 100, self.properties.getProperty('Mode'))
 
+        if self.properties.getProperty('Check status'):
+            responseChannel = 'GRASPING_STATE'
+            responseMessageClass = lcmdrc.boolean_t
+            grasping_state = lcmUtils.MessageResponseHelper(responseChannel, responseMessageClass).waitForResponse(timeout=7000)
+            if grasping_state is not None and grasping_state.data == 1:
+                self.statusMessage = "Grasping successful"
+            else:
+                self.fail("No object in hand")
+
```

Comment:
Great idea! Would this have the WaitForGraspingState task as a subtask of this one? I.e. so one property in the Grasping task can determine whether to wait or not?